docs/quickstart/quickstart.ipynb

```diff
@@ -26,14 +26,19 @@
      "name": "stderr",
```

Comment:
<p>and this is already in the notebook - so we do not need the png file - right. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
The png file doesn't have to be stored in the github repo.

Reply:
should i remove it ?

Reply:
The png file will be created each time a user runs the notebook though, but as long as the png file is not on the repo it should be fine right @wkerzendorf

Reply:
@rohitp934 - this is something that @Chinmay4400 is working on - please make your own implementation if you are interested in this project.

Reply:
@Chinmay4400 well think about what we are trying to accomplish - it is already showing it - so do we need the png?

Reply:
If a user runs the notebook, then it's fine. But if we view it in github, we might need the png file. Let me get back to you on this.

Reply:
deleting the png file breaks the image in notebook in github repo

Reply:
@wkerzendorf I deleted the image but now I can't see the image in the quickstart notebook...not without running. Makes sense as I had displayed it in markdown.

Reply:
@wkerzendorf I have updated the quickstart notebook as required. Please take a look

Reply:
<p>now make the image zoomable. </p>

Reply:
double clicking the image allows zooming. Is that not what you have in mind?